recipes/wasmedge/all/conandata.yml

```diff
@@ -1,5 +1,18 @@
 sources:
   "0.10.0":
+    # Macos:
```

Comment:
It can be removed as it's commented. Also, could replace tabs by spaces.

Reply:
I don't see tabs in this file... 🤔

Reply:
Github diff makes me crazy ... Space it uses dots, Tabs is uses arrow, but also nothing sometimes ... I thought it was mixed. Nevermid.

Please, remove commented versions. User could open issues related to a not built version but listed in conandata.